src/ergw_aaa_radius.erl

```diff
@@ -360,7 +360,13 @@ vendor_ituma('WLAN-Station-RSSI ', #{'WLAN-Station-RSSI' := Value}, Attrs) ->
 vendor_ituma('WLAN-AP-RSSI', #{'WLAN-AP-RSSI' := Value}, Attrs) ->
     [{?'IM_UE_Wwan_RSSI', Value}|Attrs];
 vendor_ituma('LI-Location', #{'CAPWAP-GPS-Latitude' := Latitude,
-			      'CAPWAP-GPS-Longitude' := Longitude}, Attrs) ->
+			      'CAPWAP-GPS-Longitude' := Longitude}, Attrs)
+	when is_float(Latitude) andalso is_float(Longitude) ->
```

Comment:
Does 0.0 is a valid latitude?

Reply:
yes, somewhere arround here https://www.rmg.co.uk/royal-observatory

Reply:
max/min is  +/-180.0, but I think the float check is good enough

Reply:
an for both being zero https://en.wikipedia.org/wiki/Null_Island